tests/fixtures/when.go

```diff
@@ -268,6 +269,40 @@ func (w *When) UpdateAutoHealConfig(autoHealEnabled bool) *When {
 	return w
 }
 
+// UpdateRepoCredentialConfig updates Numaplane controller configmap
+func (w *When) UpdateRepoCredentialConfig(config string) *When {
```

Comment:
Potentially should be renamed since this updates the entire config. May also not require the `autoHealing` function anymore.